packages/amazon/src/serverGroup/details/scalingPolicy/upsert/step/StepPolicyAction.tsx

```diff
@@ -150,7 +150,7 @@ export const StepPolicyAction = ({
       ))}
       <div className="row sp-margin-s">
         <div className="col-md-10 col-md-offset-1">
-          <button className="btn btn-block btn-sm add-new" onClick={addStep}>
+          <button type="button" className="btn btn-block btn-sm add-new" onClick={addStep}>
```

Comment:
The `TaskMonitorModal` was treating this a submit button by default so I had to be explicit with the type.